src/capturer/engine/linux/mod.rs

```diff
@@ -375,6 +375,7 @@ impl LinuxCapturer {
                 eprintln!("Error occured capturing: {e}");
             }
         }
+        CAPTURER_STATE.store(0, std::sync::atomic::Ordering::Relaxed);
```

Comment:
`STREAM_STATE_CHANGED_TO_ERROR` should also be reset to false.

Reply:
Done! Thank you for the review, (and for contributing linux support in general)!